org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/thingTranslator.js

```diff
@@ -35,13 +35,56 @@ class StateReader {
     }
 }
 
+function deviceHvacModeToTranslatorHvacMode(mode) {
+    switch (mode) {
+        case 'cool_only':
+            return 'coolOnly';
+        case 'heat_only':
+            return 'heatOnly';
+        case 'auto':
+            return 'auto';
+    }
```

Comment:
'off' in Wink is handled by the 'powered' property, rather than as part of the HVAC mode.

---
In reply to: [80141397](https://github.com/openT2T/translators/pull/101#discussion_r80141397) [](ancestors = 80141397)